WordPress/Tests/PHP/StrictInArrayUnitTest.inc

```diff
@@ -35,3 +35,4 @@ array_keys( $testing, 'my_key' ); // Warning.
 array_keys( array( '1', 1, true ), 'my_key' ); // Warning.
 array_keys( $testing, 'my_key', false ); // Warning.
 array_keys( array( '1', 1, true ), 'my_key', false ); // Warning.
+in_array( 1, array( '1', 1 ), TRUE ); // Ok.
```

Comment:
@websupporter That should be a new line **below** your code sample, not above it like you've now done. :smirk_cat: